code/modules/mob/living/carbon/human/species.dm

```diff
@@ -777,9 +777,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			if(num_legs < 2)
 				return FALSE
 			if(DIGITIGRADE in species_traits)
-				if(!disable_warning)
-					to_chat(H, "<span class='warning'>The footwear around here isn't compatible with your feet!</span>")
-				return FALSE
+				H.update_inv_shoes()
```

Comment:
Small question, does removing the check for digitigrade result in the same outcome?

Reply:
I'd honestly consider this paranoid code more than something essential, since I was having weird issues of both spawn and putting shoes on another person not updating properly.